initial-setup-reconfig.ks.in

```diff
@@ -0,0 +1,44 @@
+#test name: initial-setup-reconfig
+#
+# Check that Initial Setup is correctly configured in reconfig mode if specified in kickstart.
+%ksappend repos/default.ks
+%ksappend common/common_no_payload.ks
+
+# enable initial setup
+firstboot --enable --reconfig
+
+# make sure the initial setup package is installed
+%packages
+initial-setup
+%end
+
+%post
+# check that initial-setup RPM is installed
+rpm -q initial-setup
+if [[ $? != 0 ]]; then
+    echo '*** the initial-setup package should not be installed' >> /root/RESULT
+fi
+
+# check that initial-setup-gui RPM is not installed
+# (it was not specified in the %packages section
+#  and should not be pulled in automatically)
+rpm -q initial-setup-gui
+if [[ $? == 0 ]]; then
+    echo '*** initial-setup-gui package should not be installed' >> /root/RESULT
+fi
+
+# check that the initial-setup service is enabled
+systemctl is-enabled initial-setup
+if [[ $? != 0 ]]; then
+    echo "*** initial-setup.service is not enabled" >> /root/RESULT
+fi
+
+# the reconfig file should not be present and the --reconfig option was not
```

Comment:
I think it should be `` file should not be present an`` -> `` file should be present an``
Or maybe the whole sentence should be rewritten.

Reply:
Let's change it to:

```
# the reconfig file should only be present if the --reconfig option was
# passed to the firstboot kickstart command

```

Reply:
Looking at it now, the original text is really pretty confusing! :D